docs/gethue/content/en/posts/2022-02-01-hplsql-support.md

```diff
@@ -0,0 +1,110 @@
+---
+title: HPL/SQL Support
+author: Hue Team
+type: post
+date: 2022-02-01T00:00:00+00:00
+url: /blog/2022-02-01-hplsql-support
+sf_thumbnail_type:
+  - none
+sf_thumbnail_link_type:
+  - link_to_post
+sf_detail_type:
+  - none
+sf_page_title:
+  - 1
+sf_page_title_style:
+  - standard
+sf_no_breadcrumbs:
+  - 1
+sf_page_title_bg:
+  - none
+sf_page_title_text_style:
+  - light
+sf_background_image_size:
+  - cover
+sf_social_sharing:
+  - 1
+sf_related_articles:
+  - 1
+sf_sidebar_config:
+  - left-sidebar
+sf_left_sidebar:
+  - Sidebar-2
+sf_right_sidebar:
+  - Sidebar-1
+sf_caption_position:
+  - caption-right
+sf_remove_promo_bar:
+  - 1
+ampforwp-amp-on-off:
+  - default
+categories:
+  - Version 4.11
+  - Development
+  - Query
+
+---
+
+HPL/SQL is an Apache open source procedural extension for SQL for Hive users. It has its own grammar. It is a built-in part of Apache Hive since version 2.0.
+
+HPL/SQL is a hybrid and heterogeneous language that understands syntaxes and semantics of almost any existing procedural SQL dialect, and you can use with any database, for example, running existing Oracle PL/SQL code on Apache Hive and Microsoft SQL Server, or running Transact-SQL on Oracle, Cloudera Impala or Amazon Redshift.
+For more information about the HPL/SQL language, see the [HPL/SQL Reference](http://www.hplsql.org/doc).
+
+#### How to enable HPL/SQL dialect in Hue:
+In the desktop/conf/hue.ini config file section just add the Phoenix interpreter:
+
+    [notebook]
+    [[interpreters]]
+    [[[hplsql]]]
+    name=Hplsql
+    interface=hiveserver2
+
+  **Note:** HPL/SQL uses the [beeswax config](https://gethue.com/how-to-configure-hue-in-your-hadoop-cluster/) like Hive uses.
+
+#### HPL/SQL key features:
+- Flow of Control Statements (FOR, WHILE, IF, CASE, LOOP, LEAVE, RETURN)
+- Functions, procedures, and packages
+- Built-in functions (string manipulations, datetime functions, conversions)
+- Exception handling and conditions
+- Constants and variable, assignment (DECLARE count INT := 1)
+- Processing results using a CURSOR
+
+
+#### HPL/SQL limitations:
```

Comment:
Please remove the periods from this bulleted list for consistency between this list and the key features list.